src/Microsoft.DotNet.Build.Tasks/PackageFiles/BclRewriter.runtimeconfig.json

```diff
@@ -2,7 +2,7 @@
   "runtimeOptions": {
     "framework": {
       "name": "Microsoft.NETCore.App",
-      "version": "1.1.0"
+      "version": "2.0.0"
```

Comment:
I"m fine moving these up to 2.0.0 but we should probably make changes in Buildtools to overwrite this version with whatever we restore otherwise we will have to keep BuildTools in lock step with what CLI we are using. That can be done in another change if needed.

Reply:
Actually it cannot be done in another change because we don't want folks to not be able to consume Buildtools after your commit and before they move to the 2.0.0 SDK.

Reply:
I'll make this change as we discussed yesterday.